portafly/doc/ux_specs/audience/accounts/Listing.md

```diff
@@ -0,0 +1,56 @@
+# Audience > Accounts > Listing
```

Comment:
@cntlsn Now that we have an index, maybe it's a good idea to have a "Back to index" link in every page/component, WDYT?

Reply:
added!